foo.cmd

```diff
@@ -0,0 +1,2 @@
+erase /q packages\*.*
+dotnet pack src/Consolonia.sln -c Release -o packages --include-symbols --property WarningLevel=0
```

Comment:
is this file needed?

Reply:
wooops! Nope, I shouldn't have checked that in, that's my testing of the what the build does to make packages so I can inspect them. I will delete